plugins/ros/src/components/riScInfo/riScStatus/RiScStatusComponent.tsx

```diff
@@ -70,12 +70,74 @@ const RiScPublishDialog = ({
   );
 };
 
+interface RiScMigrationDialogProps {
+  openDialog: boolean;
+  handleCancel: () => void;
+  handleUpdate: () => void;
+}
+
+const RiScMigrationDialog = ({
+  openDialog,
+  handleCancel,
+  handleUpdate,
+}: RiScMigrationDialogProps): ReactComponentElement<any> => {
+  const { t } = useTranslationRef(pluginRiScTranslationRef);
+
+  const [saveMigration, setSaveMigration] = useState<boolean>(false);
+
+  const handleCheckboxInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSaveMigration(event.target.checked);
+  };
+
+  return (
+    <Dialog open={openDialog}>
+      <DialogTitle>{t('migrationDialog.title')}</DialogTitle>
+      <DialogContent>
+        <Alert severity="info" icon={false}>
+          <FormControlLabel
+            control={
+              <Checkbox
+                color="primary"
+                checked={saveMigration}
+                onChange={handleCheckboxInput}
+              />
+            }
+            label={t('migrationDialog.checkboxLabel')}
+          />
+        </Alert>
+      </DialogContent>
+      <DialogActions sx={dialogActions}>
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={handleUpdate}
+          disabled={!saveMigration}
+        >
+          {t('dictionary.confirm')}
+        </Button>
+        <Button variant="outlined" color="primary" onClick={handleCancel}>
```

Comment:
Button og Checkbox sin `color` verdi er primary by default, så det er overflødig

```suggestion
                checked={saveMigration}
                onChange={handleCheckboxInput}
              />
            }
            label={t('migrationDialog.checkboxLabel')}
          />
        </Alert>
      </DialogContent>
      <DialogActions sx={dialogActions}>
        <Button
          variant="contained"
          onClick={handleUpdate}
          disabled={!saveMigration}
        >
          {t('dictionary.confirm')}
        </Button>
        <Button variant="outlined" onClick={handleCancel}>
```